fem/intrules.cpp

```diff
@@ -477,6 +476,7 @@ void QuadratureFunctions1D::GaussLobatto(const int np, IntegrationRule* ir)
    */
 
    ir->SetSize(np);
+   ir->SetPointIndices();
```

Comment:
This line needs to be added still for QuadratureFunctions1D::GaussLegendre

Reply:
Thanks. It somehow fell between the cracks.